flink-yarn/src/test/java/org/apache/flink/yarn/FlinkYarnSessionCliTest.java

```diff
@@ -565,7 +565,8 @@ void testShipFiles() throws Exception {
         YarnClusterDescriptor flinkYarnDescriptor =
                 (YarnClusterDescriptor) clientFactory.createClusterDescriptor(executorConfig);
 
-        assertThat(flinkYarnDescriptor.getShipFiles()).isEqualTo(Lists.newArrayList(tmpFile));
+        assertThat(flinkYarnDescriptor.getShipFiles())
+                .isEqualTo(Lists.newArrayList(new org.apache.hadoop.fs.Path(tmpFile.toURI())));
```

Comment:
```suggestion
                .containsExactly(new org.apache.hadoop.fs.Path(tmpFile.toURI()));
```

Reply:
@RocMarshal Thanks for your review and I have updated this.